### DISCUSSION TITLE
TAG4

### DISCUSSION THREAD

**Discussion by TextAdventurer12** [ppy/osu#29164] [Ideas] [answered]
Are there any plans to port it over to lazer?

**Comment by Joehuu** [comment, marked as answer]
This is a duplicate of https://github.com/ppy/osu/discussions/27649. Please don't forget to upvote!

There are plans to bring most stable features back, but as things stand, as an external contributor standpoint, the focus seems to be on new and exclusive features right now and there are not enough devs to balance both.
